user-interface/src/main/java/life/qbic/datamanager/views/projects/purchase/PurchaseItemDeletionConfirmationNotification.java

```diff
@@ -0,0 +1,31 @@
+package life.qbic.datamanager.views.projects.purchase;
+
+import com.vaadin.flow.component.Text;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import life.qbic.datamanager.views.notifications.NotificationDialog;
+
+/**
+ * Warns the user that the file will be deleted from the server.
+ * <p>
+ * This dialog is to be shown when a user wants to delete an Offer item.
+ */
+public class PurchaseItemDeletionConfirmationNotification extends NotificationDialog {
+
+  public PurchaseItemDeletionConfirmationNotification() {
+    addClassName("batch-deletion-confirmation");
+    customizeHeader();
+    content.add(new Div(new Text(
+        "Are you sure you want to delete this file?")));
```

Comment:
NIT: We could be more specific here. Although maybe it also makes sense to include the name of the offer to be deleted in the confirmationdialog?
```suggestion
        "Are you sure you want to delete this offer?")));
```